new-client/src/plugins/vtsearch/SearchModel.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * @summary SearchModel used for VT specific searches.
+ * @description NEED TO ADD A DESCRIPTION
+ *
+ * @class SearchModel
+ */
+export default class SearchModel {
+  constructor(settings) {
+    this.map = settings.map;
+    this.app = settings.app;
+    this.localObserver = settings.localObserver;
+  }
+
+  /**
+   * Adjusts a WKT so that it's supported for a web browser and GeoServer.
+   * @param wkt The wkt that needs to be adjusted.
+   * @returns Returns a supported wkt for GeoServer.
+   *
+   * @memberof SerachModel
+   */
+  fixWktForGeoServer = wkt => {
+    return wkt
+      .replace(/\(/g, "%28")
+      .replace(/\)/g, "%29")
+      .replace(/\ /g, "%20")
+      .replace(/\,/g, "%5C,");
+  };
+
+  /**
+   * Gets requested journeys.
+   * @param fromTime Start time.
+   * @param endTime End time.
+   * @param wktPolygon A polygon as a WKT.
+   * @returns Returns all journeys
+   *
+   * @memberof SearchModel
+   */
+  getJourneys(filterOnFromDate, filterOnToDate, filterOnWkt) {
+    this.localObserver.publish("journey-result-begin");
+
+    // Fix parentheses and so on, so that the WKT are geoserver valid.
+    filterOnWkt = this.fixWktForGeoServer(filterOnWkt);
+
+    // Build up the url.
+    let url =
+      "http://sestoas256:8080/geoserver/kartsidan2/ows?service=WFS&version=1.0.0&request=GetFeature&typeName=kartsidan2:journeys&outputFormat=application/json";
+    const viewParams = `&viewparams=filterOnFromDate:${filterOnFromDate};filterOnToDate:${filterOnToDate};filterOnWkt:${filterOnWkt}`;
+    url = url + viewParams;
```

Comment:
Any changes that you'll lift the `url` param out instead of the hard-coded value? I understand it's early in the process, but we've unfortunately seen stuff like this being forgotten later on (as it "just works" on the developer's machine), and not being fixed upon release.

Reply:
I agree on your comment, I have seen this myself over the years as well. 
In this particular case the reason for this bad code is that a colleague will need this values soon. The risk that this will be forgotten is equals to zero because the next step is to add more functions and fix the urls to not be hard-coded.

Reply:
OK, let's merge then.